models/users.js

```diff
@@ -306,6 +309,78 @@ const initializeUser = async (userId) => {
   return true;
 };
 
+/**
+ * Adds user data for verification by moderators
+ * @param userId {String} - RDS User Id
+ * @param discordId {Object} - Discord id of RDS user
+ * @param profileImageUrl {number} - profile image URL of user
+ * @param discordImageUrl {number} - discord image URL of user
```

Comment:
can you also add the return type and error it throws in the definition

Reply:
done